Equation.java

```diff
@@ -0,0 +1,49 @@
+import static java.lang.Math.*;
+/**
+ * solve the quadratic equation
+ * @author Babak Alexey
+ */
+public class Equation {
+
+   /** 
+    * check possibility of the "String -> Double" conversion
+    * @param string  the string to check,
+    * if string has only double variable  - return true
+    */
+   public static boolean checkString(String string) {
+      try {
+         Double.parseDouble(string);
+      } catch (Exception e) {
+           return false;
+      }
+      return true;
+   }
+
+   /** 
+    * solve the quadratic equation 
+    */
+   public static void main(String [] args) {
+      if (args.length != 3) {
+         System.out.println("Try to input 3 numbers ( a,b,c)");
+         System.exit(0);

```

Comment:
To indicate that your application finished with error here must be not zero exit code